docs/docs/building-chatbots/memory-&-data-persistence/flow-memory.md

```diff
@@ -5,43 +5,39 @@ title: Flow Memory
 
 --------------------
 
-In a conversation, you may want to ask questions to the user and remember his answers to use them later. You may also want to access the values of system parameters, such as the values of the slots that your chatbot just extracted.
+In a conversation, you may want to ask questions to the user and remember their answers for later use. You may also want to access the values that your chatbot extracts.
 
 ## System Parameters
 
-When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you will see the tree of all the system parameters that it tracks. Just click the debugger button (circled in red below) and it will open in the bottom panel by default.
+When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you will see a tree of all tracked system parameters. Click the debugger button in the top right corner and it will open in the bottom panel by default.
```

Comment:
```suggestion
When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you will see a tree of all tracked system parameters. Click the debugger iconin the top right corner and it will open in the bottom panel by default.
```

button or icon?